democracy/tests/test_hearing.py

```diff
@@ -214,6 +215,25 @@ def test_8_get_detail_labels(api_client):
 
 
 @pytest.mark.django_db
+def test_8_get_detail_organization(api_client):
+    hearing = Hearing()
+    hearing.save()
+
+    organization = Organization(name='The department for squirrel welfare')
+    organization.save()
```

Comment:
You could use pytest fixture for hearing here, there is `default_hearing` for example. Also organization should be made as a fixture.